### DISCUSSION TITLE
reactV6，引入antd mobile里的Input输入框，如何在获取焦点时显示边框？

### DISCUSSION THREAD

**Discussion by ken428965** [ant-design/ant-design-mobile#5835] [Q&A]
**reactV6，引入antd mobileV5.22里的Input输入框，希望对Input输入框获取焦点时显示其边框
代码如下：**
```
<Input placeholder='请输入用户名' onFocus={()=>{style={'--border' : 'solid 1px #cccccc'}}}/>
```
**目前效果是，点击输入框时没有显示出边框**

**试过找过很多关于使用input和onfocus的文章，然而没有找到解决办法
请问代码出了什么问题？正确的写法是什么？请展示代码举例说明，谢谢。**

**Comment by dogeow** [comment]
onFocus 是对应原生属性的，是写 JS 的，你返回对象没用吧？

我是一个后端，本着分享的原则发表评论，如果我说的不对，请见谅。

**Reply by ken428965** [reply]
感谢评论和分享~
您说得很对，它是对应原生属性的，返回对象会无效的
